src/Engines/MySql/Sandbox/Sandbox.php

```diff
@@ -29,23 +29,25 @@
 
 class Sandbox
 {
+    const DEFAULT_ENGINE = 'MySQL';
+    const DEFAULT_ENGINE_VERSION = '8.0.28';
```

Comment:
This seems really specific. Will it have to change a lot over time?

Reply:
I see your point. I decided to remove default version from code. The default would now be AWS RDS default (as previously).